cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -69,6 +69,11 @@ def filtered_list(cls):
         if not settings.FEATURES.get('ENABLE_MOBILE_SOCIAL_FACEBOOK_FEATURES'):
             filtered_list.append('facebook_url')
 
+        # Do not show social sharing url field if the feature is disabled.
+        if not settings.FEATURES.get('DASHBOARD_SHARE_SETTINGS') or \
```

Comment:
Another crazy nit, and maybe I'm in the wrong here, but rather than a newline, use parenthesis?